components/Ripple/src/MDCRippleView.m

```diff
@@ -60,6 +60,15 @@ - (void)commonMDCRippleViewInit {
 - (void)layoutSubviews {
   [super layoutSubviews];
   [self updateRippleStyle];
+  self.frame = self.superview.bounds;
+}
+
+- (void)layoutSublayersOfLayer:(CALayer *)layer {
+  [super layoutSublayersOfLayer:layer];
+  for (CALayer *sublayer in self.layer.sublayers) {
+    sublayer.frame = self.bounds;
```

Comment:
Call `CGRectStandardize` before assigning the `bounds` to the sublayer's `frame`.

Reply:
done